keymirror/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for keymirror 0.1.1
+// Type definitions for keymirror 0.2.0
+// TypeScript Version: 2.1
```

Comment:
This belongs at the *last* line of the header, after `// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`.